stratum/hal/lib/tdi/es2k/es2k_tdi_status_test.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "gtest/gtest.h"
+#include "ipu_types/ipu_types.h"
+#include "stratum/hal/lib/tdi/tdi_bf_status.h"
+
+namespace stratum {
+namespace hal {
+namespace tdi {
+
+class Es2kTdiStatusTest : public ::testing::Test {};
+
+// Verify that IPU status codes map directly to TDI status codes.
```

Comment:
Like this unit test to verify that the error codes map directly

Reply:
Thanks!